scripts/generate_documentation_list.js

```diff
@@ -51,6 +51,16 @@ function isDirectory(source) {
   return fs.lstatSync(source).isDirectory();
 }
 
+/**
+ * @param {string} val
+ * @returns {string}
+ */
+function encodeHtmlAttributeValue(val) {
```

Comment:
This is the same function as the other change in `scripts/generate_demo_list.js`, maybe we should add it in a utils.js in the scripts directory to avoid redundancy

Reply:
As I replaced all `encodeHtmlAttributeValue` calls by `html-entities`'s `encode` method, this question should not be necessary anymore.